profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestUsage.java

```diff
@@ -42,6 +42,8 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
@reshmabidikar We're trying to move away from Guava. @xsalefter can help with the transition if needed, although I don't think the import is needed.

Reply:
Yes, on this file, `ImmutableMap` is not needed, so perhaps just remove it? 

On other files, we're migrating guava's collection to Java's stream API, as described in issue #1615. But if it's too much for you, just leave it as is. Currently, I'm working on `killbill-invoice` module, so naturally you would see a lot of `import com.google.common.<XXX>` statement in `killbill-jaxrs`.